CHANGELOG.md

```diff
@@ -43,6 +43,8 @@
 - Added support for applying Snowpark Python function `snowflake_cortex_summarize`.
 - Added support for `DataFrame.attrs` and `Series.attrs`.
 - Added support for `DataFrame.style`.
+- Added support for `DataFrame`/`Series.attrs`
```

Comment:
Should this be in this PR?

Reply:
seems like a merge artifact (I think #2503 changed the wording on the `attrs` entry)

Reply:
Yeah that was added from the merge, I've now removed it